apps/tailwind-components/app/pages/Form.story.vue

```diff
@@ -204,8 +204,9 @@ const numberOfFieldsWithErrors = computed(
           </Button>
           <FormFields
             class="grow"
-            :columns="visibleColumns"
+            :columns="metadata.columns"
             :errorMap="errorMap"
+            :visibleMap="visibleMap"
```

Comment:
passing the both a struct and part of het same struct via the prosp makes it complicated ( more complicated than needed ? )